cmd/kubelet/app/server.go

```diff
@@ -625,6 +625,7 @@ func CreateAPIServerClientConfig(s *options.KubeletServer) (*restclient.Config,
 		return nil, err
 	}
 
+	clientConfig.UserAgent = restclient.UserAgentForComponent(componentKubelet)
```

Comment:
a comment please "hard-code instead of using os.Args[0] because ...."

Reply:
done.